fares-data-build-tool/create-data
# feat: Cap pricing per distance page

## Description

We need a new page which the user is redirected to after they select pricing per distance on the typeOfCap page.

## Testing instructions

Follow capped journey and select type of cap to be by distance then you should see this page and test input combinations.
